### ISSUE TITLE
Add releases

### ISSUE THREAD

**Issue by flauntingspade4** [drodil/rustybeer#83]
Few people will have rustc on their system, and that may put them off using the CLI-the project could be built with the `--release` tag and put in the `releases` tab of the main page to make it more accessible for people to download and use in their brewing projects

**Comment by drodil** [comment]
Agree. But we might want to hold on for a bit longer to have more content. Also publishing to crates.io should be part of the releasing
